Automattic/jetpack
# Social: Update/cut component tooltip

Duplicate of #29224
Fixes # 1203876111187910-as-1203884651474282

## Proposed changes:

* Updated the CSS rules in the ContextualUpgradeTrigger component and changed the text copy.
 
### Other information:


## Jetpack product discussion



1203876111187910-as-1203884651474282

## Does this pull request change what data or activity we track or use?



No.

## Testing instructions:






* Boot up a new JN site on this branch with Social and Protect plugins. Boot up a trunk site on this branch with Social and Protect plugins
* Make sure you have the free plans on both and compare the behaviour of the upgrade box by hovering over them. You will also see the information icon in this branch. Hover over it to see the upgrade copy.  

Before

<img width="703" alt="Screenshot 2023-03-01 at 19 02 00" src="https://user-images.githubusercontent.com/6594561/222153284-79f221d1-61fa-4b5e-88c8-7fe2ea599fb8.png">

After

<img width="697" alt="Screenshot 2023-03-01 at 18 15 01" src="https://user-images.githubusercontent.com/6594561/222143047-59db8651-8ac2-4d8d-8f5e-781716272564.png">

* Testing the component in another plugin: Install Jetpack Protect on your JN site and make sure the upgrade copy looks the same. 


Before and After:

<img width="641" alt="Screenshot 2023-03-01 at 18 59 40" src="https://user-images.githubusercontent.com/6594561/222153077-0d6f8255-ead7-4a31-b360-63e65f900977.png">
is is to ensure that our changes to the `ContextualUpgradeTrigger` component didn't break anything.
